### DISCUSSION TITLE
Code Sample for reading excel files

### DISCUSSION THREAD

**Discussion by franks1** [paillave/Etl.Net#375] [Q&A] [answered]
I have gone through the sample and tutorial of your extraordinary Library  (Etl.Net). I would like you to assist me with a sample code on reading Excel Files. Thank you

**Comment by paillave** [comment, marked as answer]
Created an issue out of this.
https://github.com/paillave/Etl.Net/issues/376
I will make it asap

**Reply by franks1** [reply]
Ok. Thank you

**Reply by paillave** [reply]
You will find a sample on the issue.
Note: you must have the latest version v2.0.47 for it to work with this syntax.

**Comment by franks1** [comment]
Thanks, I'll check it out.

**Comment by franks1** [comment]
Thank you
